### PR TITLE
add bookmark page with the current active class and linking between h…

### PR DESCRIPTION
**Pull request by MadinaKon**
…ome and bookmark pages

### CODE REVIEW

File: bookmark.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Homepage</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.4.1/dist/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
```
Comment (danielrileyde) [inline bookmark.html:7]: Are you using bootstrap here?

File: css/styles.css [hunk 2]
```diff
@@ -29,21 +29,23 @@ a {
...
   height: 60vh;
-  background: teal;
+
   flex: 1;
+  display: flex;
+  flex-direction: column;
+  gap: 50px;
+  /* margin: 80px; 
+   background: teal;
+  */
```
Comment (dittley) [inline css/styles.css:39]: Does this need to stay here, if it's not being used?

File: bookmark.html (hunk 1 above)
Comment (dittley) [inline bookmark.html:7]: Why do you need bootstrap here?

**Review by dittley — COMMENTED** [review]
I don't see any problems with this code, so happy merging. 

Next time on your commit message, give some background on Laya's feedback, I don't know what she recommended, so it's hard to see what has been improved.

**Pull request merged**